### ISSUE TITLE
Kernel 4.18/4.19/4.20 and >= 5.0 fix

### ISSUE THREAD

**Issue by EasyNetDev** [dorimanx/exfat-nofuse#148]
Hi all,

Because I see this source-code is not maintained anymore, I've forked to my Github and I've added the patch for kernel 4.18/4.19/4.20 and also for 5.0.
Please take a look here: https://github.com/EasyNetDev/exfat-nofuse

Kind regards,
Adrian

**Comment by EasyNetDev** [comment]
Update: added support for Debian DKMS packaging from https://github.com/eugenesan/exfat-nofuse and I've adapted a little bit.

**Comment by warrenc5** [comment]
Thank you so much!

**Comment by re2zero** [comment]
That's  great job!

**Comment by EasyNetDev** [comment]
Anyway, this driver is included in Linux main kernel for a while.

**Comment by re2zero** [comment]
yes, but I meet some problems while try to back port it to 4.19, and I don't know how to fix them, because they are  using some new features base on main kernel, that are  fs_context.h and fs_parser.h related.

**Comment by EasyNetDev** [comment]
Hi @re2zero,

Indeed, you have right. For such old kernels you need something that is working. For sure my fork is working. I was using on my desktop at that time for long time until kernel 5.4 when Samsung decided to push exFAT to mainline.
Hope you find it right for your project.
